File: app/src/main/java/com/example/jacob/myapplication/Activities/PersonActivity.java
package com.example.jacob.myapplication.Activities;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.util.Log;
import android.widget.TextView;

import com.example.jacob.myapplication.Constants;
import com.example.jacob.myapplication.R;

public class PersonActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_person);
        Log.i("Persona", Constants.person.toString());
        TextView aggrTV = (TextView) findViewById(R.id.person_view_aggressiveness);
        aggrTV.setText(Constants.person.getAggresiveness() * 100 + "%");
        TextView happTV = (TextView) findViewById(R.id.person_view_happiness);
        happTV.setText(Constants.person.getHappiness()*100 + "%");
        TextView fearTV = (TextView) findViewById(R.id.person_view_fear);
        fearTV.setText(Constants.person.getFear()*100 + "%");
        TextView loveTV = (TextView) findViewById(R.id.person_view_love);
        loveTV.setText(Constants.person.getLove()*100 + "%");
        TextView sadTV = (TextView) findViewById(R.id.person_view_sadness);
        sadTV.setText(Constants.person.getSadness()*100 + "%");
    }
}

File: app/src/main/java/com/example/jacob/myapplication/activities/PersonActivity.kt
package com.example.jacob.myapplication.activities

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.util.Log

import com.example.jacob.myapplication.Constants
import com.example.jacob.myapplication.R

import java.util.Locale

import kotlinx.android.synthetic.main.activity_person.*

class PersonActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_person)

        Log.i("Persona", Constants.person.toString())
        person_view_aggressiveness.text = getFormattedString(Constants.person.aggressiveness)
        person_view_happiness.text = getFormattedString(Constants.person.happiness)
        person_view_fear.text = getFormattedString(Constants.person.fear)
        person_view_love.text = getFormattedString(Constants.person.love)
        person_view_sadness.text = getFormattedString(Constants.person.sadness)
    }

    private fun getFormattedString(param: Float): String {
        return String.format(Locale.getDefault(), "%.0f%%", param * 100)
    }
}
